src/main.py

```diff
@@ -0,0 +1,102 @@
+from AlertsAggregator import AlertsAggregator
+from OrefAlertsRetrieve import OrefAlertsRetrieveData  
+from fastapi import FastAPI, HTTPException 
+import pandas as pd
+import requests
+from Errors import RetrieveDataException, WrongSettlementException
+from models import AlertsQueryInput
+from datetime import datetime, timedelta
+app = FastAPI()
+
+CSV_FILE_PATH = "C:/Projects/Python/Projects/BestTimeToShower/data/data.csv"
+dataframe = pd.read_csv(CSV_FILE_PATH)
+getdata = OrefAlertsRetrieveData()
+alert_aggregator = AlertsAggregator(dataframe)
+
+
+
+@app.post("/orefalerts")
+async def get_oref_alert(from_date: str, to_date: str):
+    try:
+        global alert_aggregator
+        getdata.get_oref_alarms(from_date, to_date)
+        df = pd.read_csv(CSV_FILE_PATH)
+        alert_aggregator = AlertsAggregator(df)
+        return {"message": "Up-date the alerts data csv"}
+    except RetrieveDataException as e:
+        raise HTTPException(status_code=404, detail=str(e))
+    except PermissionError as ex:
+        raise HTTPException(status_code=401, detail=str(ex))
+    except Exception as exc:
+        raise HTTPException(status_code=500, detail=str(exc))
```

Comment:
This is a UNKOWN ERROR!
add a matching prefix. do this all over  


```suggestion
        raise HTTPException(status_code=500, f"Unkown Error: details: {exc}")
```